stable/spinnaker/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Open source, multi-cloud continuous delivery platform for releasing software changes with high velocity and confidence.
 name: spinnaker
-version: 1.16.1
+version: 1.19.0
```

Comment:
Why this big bump? @afcastano @paulczar 
Should we keep this more or less in sync with the spinnaker version (eg. 1.16.1?)